product_docs/docs/pem/8/registering_database_server.mdx

```diff
@@ -93,11 +93,11 @@ pemworker --register-server \
           --server-user prod_admin 
 ```
 
-In the following example, we specify different parameters to be used for the connection from the agent to the monitored server.
-We override the fully qualified domain name provided in `--server-addr` with `localhost` instead and override the user to use a `local_monitor` user. 
-This could be a user which is only permitted to connect from the same host for example.
+This example specifies different parameters to use for the connection from the agent to the monitored server.
+The example overrides the fully qualified domain name provided in `--server-addr` with `localhost` instead and overrides the user to use a `local_monitor` user. 
+This might be a user who's permitted to connect only from the same host for example.
```

Comment:
Postgres users are not necessarily people, so I'm not sure 'who' is appropriate. Is there a precedent elsewhere?

Reply:
Oh, interesting. The "which" was wrong in any case, so I'll change it to this:
```suggestion
This might be a user that can connect only from the same host, for example.
```